src/lib.rs

```diff
@@ -61,6 +61,8 @@ pub mod uart;
 ))]
 pub mod ulp;
 pub mod units;
+#[cfg(esp_idf_soc_usb_serial_jtag_supported)]
```

Comment:
Can we rename `usb_serial` to just `jtag` (for the module, for the peripheral name and - in future - for the driver)? I think this is a shorter and simpler name. It is clear that jtag is exposed via usb anyway.